src/external/views/nunjucks/view-licences/macros/point-string.njk

```diff
@@ -0,0 +1,3 @@
+{% macro pointString(point) %}
```

Comment:
I assume the NGRs are always 8 char?

Reply:
I just had a look at the handlebars filter that I was recreating and realised that it's done differently there, will update it to match.